src/panel/layouts.js

```diff
@@ -25,6 +26,31 @@ export function getMapPaddings({ isMobile, isDirectionsActive }) {
   return MOBILE_CARD;
 }
 
+export function getVisibleBbox() {
+
+  const bbox = window.map.mb.getBounds();
+  let ne = bbox.getNorthEast();
+  let sw = bbox.getSouthWest();
+  const ne_canvas = window.map.mb.project(ne);
+  const sw_canvas = window.map.mb.project(sw);
+
+  if (isMobileDevice()) {
+    // On mobile, compute a bbox that excludes the resizable panel height and the header's height
+    ne_canvas.y += 65;
+    sw_canvas.y -= window.innerHeight / 2;
```

Comment:
Are we sure that the bbox should completely exclude the bottom half of the screen, including when the panel is reduced ?

Reply:
Well, it's debatable indeed: on principle yes, but it fits in a slightly larger edge case:

When we open the category, the panel is half-open and we want the request to target the upper half of the map.

If we move the panel to the bottom, what should happen? a new request with the updated bbox?

If we move the map with the panel folded at the bottom, should we recompute the list with the big bbox?

If we move the panel from bottom to the half of the screen, should we recompute the results with the small bbox?

If the answer to these questions is yes, then we can indeed take the panel folding into account in the bbox computation, but we also need to trigger new search requests when the panel is resized, which imo could look confusing, but we can try.

And if we decide to do it, I wonder what is the best way to "measure"  the panel height and send it to getVisibleBbox. maybe @bbecquet has an idea to do it "clean" ?

Reply:
We'll only exclude the header from the search bbox finally :)